selectel/mks.go

```diff
@@ -57,7 +57,8 @@ func getMKSClusterV1Endpoint(region string) (endpoint string) {
 }
 
 func waitForMKSClusterV1ActiveState(
-	ctx context.Context, client *v1.ServiceClient, clusterID string, timeout time.Duration) error {
+	ctx context.Context, client *v1.ServiceClient, clusterID string, timeout time.Duration,
+) error {
```

Comment:
We do not need this line break here and below. Please roll back these changes.

Reply:
formating changes for mks and dbaas related files are reset